src/__tests__/parser.ts

```diff
@@ -986,6 +984,39 @@ describe('parser', () => {
         });
       });
 
+      it('should allow filtering by parent interfaces', () => {
```

Comment:
Maybe it could make sense to add a test that asserts all `parents` as well. Right now from the code, it's not obvious what parents really are.

Reply:
Good point 👍